packages/compass-web/src/entrypoint.tsx

```diff
@@ -296,6 +294,21 @@ const CompassWeb = ({
     preferences: preferencesAccess.current,
   });
 
+  useEffect(() => {
+    const updateEarlyIndexesPreferences = async () => {
+      await preferencesAccess.current.savePreferences({
```

Comment:
Nice, can we add a comment linking this jira ticket so future readers have a bit more context?
```suggestion
    // TODO(COMPASS-9353): Provide a standard way of updating Compass' preferences from web.
    // Avoid duplicating this pattern until we address this ticket.
    const updateEarlyIndexesPreferences = async () => {
      await preferencesAccess.current.savePreferences({
```